audio/src/main.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import threading
+import json
+from time import sleep
+import traceback
+from commandhandler import CommandHandler
+from coreconnection import CoreConnection
+
+
+def run_cli(cmd_handler):
```

Comment:
This is just for testing/debugging, right? It fills the same niche as dummy_core, would it be a lot of work to add support for prepared sequences to dummy_core instead? Then we could debug all components in the same way.

Reply:
Ah, true! It's mostly a relic from my early testing but moving it to dummy_core instead is better!